Readme.md

```diff
@@ -68,6 +68,7 @@ memory_limit=128M
 | Memcached  | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcached.so`   |
 | MongoDB    | `${bref:extra.mongodb-php-74}`   | `extension=/opt/bref-extra/mongodb.so`     |
 | MsgPack    | `${bref:extra.msgpack-php-74}`   | `extension=/opt/bref-extra/msgpack.so`     |
+| Newrelic    | `${bref:extra.newrelic-php-74}`   | `extension=/opt/bref-extra/newrelic.so`     |
```

Comment:
"`extension=/opt/bref-extra/newrelic.so`" should not be needed

Reply:
I mirrored the other entries in the table. Is there a reason to not fill it in on the newrelic row?

Reply:
Yes, because you add it in the layer. See https://github.com/brefphp/extra-php-extensions/pull/92#discussion_r534513241